tabs/src/components/BeaverHours/Scheduler.tsx

```diff
@@ -0,0 +1,158 @@
+import { createContext, useContext, useState } from "react";
+
+const ScheduleContext = createContext({
+  state: {} as OfficeHoursSchedule,
+  updateSchedule: (newSchedule: OfficeHoursSchedule) => {},
+});
+
+export function Scheduler() {
+  return <ScheduleForm />;
+}
```

Comment:
This isn't really doing too much! Might as well just export the Schedule Form : ).